ydb/library/yql/tools/pgrun/pgrun.cpp

```diff
@@ -804,6 +812,64 @@ std::string FormatCell(const TString& data, const TColumn& column, size_t index,
     return fmt::format("{0}{1:<{2}}", delim, data, column.Width);
 }
 
+// from postgres/src/include/utils/builtins.h
+extern "C" ui64 hex_encode(const char *src, size_t len, char *dst);
```

Comment:
explicit header?

Reply:
or, better, add such function to pg_wrapper/interface/utils

Reply:
> explicit header?

original PG's header pulls in tons of PG staff we don't need in pgrun

Reply:
> or, better, add such function to pg_wrapper/interface/utils

You mean moving the declaration of hex_encode() into pg_wrapper/interface/utils.h ?